clients/admin-ui/src/features/subject-request/ActivityTimeline.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Flex, Text } from '@fidesui/react';
+import React from 'react';
+
+import { GreenCheckCircle } from '../common/Icon';
+import { PrivacyRequest } from '../privacy-requests/types';
+
+type ActivityTimelineProps = {
+  subjectRequest: PrivacyRequest;
+  setEventDetails: Function;
+};
+
+const ActivityTimeline = ({
+  subjectRequest,
+  setEventDetails,
+}: ActivityTimelineProps) => {
+  const { results } = subjectRequest;
+
+  const resultKeys = [
+    'postgres_example',
```

Comment:
Is this test data?

Reply:
Yes, this is for a component that isn't being displayed yet. It will be "enabled" in a future PR. Sean and I agreed to merge in this version of it now and update in the future.